### ISSUE TITLE
Large doors frequently throw errors about pathing

### ISSUE THREAD

**Issue by TheRealNin** [TheRealNin/SiegePlusPlus#13]
If you look at nav_debug, you'll often see that the pathing squares that are entirely within the obstacle circle are often not excluded properly. I *highly* suspect that NS2's pathing algo doesn't like obstacles that fill an entire voxel, so I suspect that if we make the front doors add a series of smaller obstacles, this wouldn't happen. This would fix all sorts of pathing problems we have with doors.

**Comment by DalinSeivewright** [comment]
Yeah.  I can confirm that if you take a regular siege or front door, cut its width in half and use two doors instead everything works fine.  Samoose was asking me to provide more screenshots of the issue from different perspectives but I just haven't gotten to it yet.

**Comment by DalinSeivewright** [comment]
This was due to NS2s pathing engine.  We have since removed ObstacleMixin from a lot of entities which fixed this issue.  Please close @TheRealNin ?
